account_fiscal_position_rule/models/account_fiscal_position_rule.py

```diff
@@ -0,0 +1,326 @@
+        comodel_name='account.fiscal.position.template',
+        string='Fiscal Position',
+        required=True
+    )
+    use_sale = fields.Boolean(
+        string='Use in sales order'
+    )
+    use_invoice = fields.Boolean(
+        string='Use in Invoices'
+    )
+    use_purchase = fields.Boolean(
+        string='Use in Purchases'
+    )
+    use_picking = fields.Boolean(
+        string='Use in Picking'
+    )
+    date_start = fields.Date(
+        string='Start Date',
+        help="Starting date for this rule to be valid."
+    )
+    date_end = fields.Date(
+        string='End Date',
+        help="Ending date for this rule to be valid."
+    )
+    sequence = fields.Integer(
+        string='Priority',
+        required=True,
+        default=10,
+        help='The lowest number will be applied.'
+    )
+    vat_rule = fields.Selection(
+        selection=[('with', 'With VAT number'),
+                   ('both', 'With or Without VAT number'),
+                   ('without', 'Without VAT number')],
+        string="VAT Rule",
+        default='both',
+        help=('Choose if the customer need to have the'
+              ' field VAT fill for using this fiscal position')
+    )
+
+
+class WizardAccountFiscalPositionRule(models.TransientModel):
+    _name = 'wizard.account.fiscal.position.rule'
+    _description = 'Account Fiscal Position Rule Wizard'
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        string='Company',
+        required=True,
+        default=lambda self: self.env['res.company']._company_default_get(
+            'wizard.account.fiscal.position.rule'))
+
+    @api.multi
+    def _template_vals(self, template, company_id, fiscal_position_id):
+        return {
+            'name': template.name,
+            'description': template.description,
+            'from_country': template.from_country.id,
+            'from_state': template.from_state.id,
+            'to_invoice_country': template.to_invoice_country.id,
+            'to_invoice_state': template.to_invoice_state.id,
+            'to_shipping_country': template.to_shipping_country.id,
+            'to_shipping_state': template.to_shipping_state.id,
+            'company_id': company_id,
+            'fiscal_position_id': fiscal_position_id,
+            'use_sale': template.use_sale,
+            'use_invoice': template.use_invoice,
+            'use_purchase': template.use_purchase,
+            'use_picking': template.use_picking,
+            'date_start': template.date_start,
+            'date_end': template.date_end,
+            'sequence': template.sequence,
+            'vat_rule': template.vat_rule,
+        }
+
+    @api.multi
+    def action_create(self):
+        obj_fpr_temp = self.env['account.fiscal.position.rule.template']
+        company_id = self.company_id.id
+
+        fsc_rule_template = obj_fpr_temp.search([])
+        # TODO fix me doesn't work multi template that have empty fiscal
```

Comment:
I'm not fond of improvement requests while migrating a module, but what's the current status of this TODO?
If it's still needed, do you plan to work on it?